src/plugins/pcr/src/InSilicoPcrWorkflowTask.cpp

```diff
@@ -37,7 +39,9 @@ QList<Task*> InSilicoPcrWorkflowTask::onSubTaskFinished(Task* subTask) {
 
     if (subTask == pcrTask) {
         foreach (const InSilicoPcrProduct& product, pcrTask->getResults()) {
-            auto productTask = new ExtractProductTask(product, productSettings);
+            QVariantMap hints;
+            hints[DocumentRemovalMode_Synchronous] = QString();
```

Comment:
nit: use `true`, not `QString()` like for other boolean-like hints

Reply:
OK